src/main/java/com/lberkholtz/advancedava/week3/service/BasicStockService.java

```diff
@@ -14,10 +18,10 @@ public BasicStockService() {
     }
 
     /**
- * Create a new <CODE>BasicStockService</CODE> instance
- *
- * @param stockService used to get actual stock data (external dependency)
- */
+    * Create a new BasicStockService instance
+    *
+    * @param stockService used to get the stockservice
+    */
 
 public BasicStockService(StockService stockService){
```

Comment:
I don't understand this. why are you passing in a StockService to the BasicStockService constructor?

Reply:
Good question! Makes no sense at all. I fixed it.